src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
-// write code here
+function convertToNumber(value) {
+  return Number(value);
+}
+
+// Функція для сортування списку за зарплатою в порядку спадання
+// eslint-disable-next-line no-shadow
+function sortList(list) {
+  // Отримуємо всі елементи списку
+  const items = [...list.children];
+
+  // Сортуємо елементи за зарплатою в порядку спадання
+  items.sort((a, b) => {
+    const salaryA = convertToNumber(a.dataset.salary);
+    const salaryB = convertToNumber(b.dataset.salary);
+
+    return salaryB - salaryA; // Спадання
+  });
+
+  // Додаємо відсортовані елементи назад в список
+  items.forEach((item) => list.appendChild(item));
+}
+
+// Функція для отримання масиву співробітників
+// eslint-disable-next-line no-shadow
+function getEmployees(list) {
```

Comment:
The `getEmployees` function is correctly defined to return an array of employee objects. Ensure that the `list` parameter is a valid DOM element with children having the necessary data attributes (`data-name`, `data-position`, `data-salary`, `data-age`).